examples/java/webapis/src/main/java/org/apache/beam/examples/webapis/UsingHttpClientExample.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.examples.webapis;
```

Comment:
Minor nit: in the future, it would be helpful to split PRs like this in 2; it makes it easier to review (and likely would drive down the time to review completion since it requires less time carved out)

Reply:
The code example is necessary for the text in the website. The second example builds on the first. The first is not really valuable. The second is.

Reply:
Yep, in this case we'd be splitting up the pr not to deliver incremental value, but because smaller PRs are easier to review and generally safer (safety is a little less important for examples) see go/small-cls or https://blog.codacy.com/small-pull-requests